app/models/support_interface/applications_filter.rb

```diff
@@ -131,6 +135,22 @@ def interviews_filter
       }
     end
 
+    def subject_filter
+      subject_options = MinisterialReport::SUBJECTS.map do |subject, _|
```

Comment:
is it time to move `MinisterialReport::SUBJECTS` into something reflecting what it actually is?